recipes/perl-biox-workflow/meta.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: perl-biox-workflow
+  version: "0.27"
+
+source:
+  fn: BioX-Workflow-0.27.tar.gz
+  url: https://cpan.metacpan.org/authors/id/J/JI/JILLROWE/BioX-Workflow-0.27.tar.gz
+  md5: ac263156ed1668265712c81c3199644f
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - perl
```

Comment:
perl-threaded is a special version of perl, compiled with threading support. Please don't depend on both perl and perl-threaded. Just on perl-threaded. Thanks!